torch/onnx/symbolic.py

```diff
@@ -1060,6 +1060,16 @@ def _cast_func_template(to_i, g, input, non_blocking):
     name = '_cast_{}'.format(k)
     globals()[name] = parse_args('v', 'i')(partial(_cast_func_template, v))
 
+pytorch_type_to_scalar_type = [
```

Comment:
I think hardcode here is fine, but please add some comments to explain why it's fine :-)